schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/IFlushRequest.java

```diff
@@ -19,10 +19,12 @@
 package org.apache.iotdb.lsm.request;
 
 /** Represents a flush request that can be processed by the lsm framework */
-public interface IFlushRequest<K, V> extends IRequest<K, V> {
+public interface IFlushRequest<K, V, M> extends IRequest<K, V> {
 
   @Override
   default RequestType getRequestType() {
     return RequestType.FLUSH;
   }
+
+  M getRootMemNode();
```

Comment:
Here you can change IFlushRequest<K, V, M> to IFlushRequest<K, V>;
Change M getRootMemNode() to <M extends MemManager> M getRootMemNode();